packages/forms-web-app/src/config.js

```diff
@@ -45,5 +45,6 @@ module.exports = {
     port: Number(process.env.PORT || 3000),
     sessionSecret: process.env.SESSION_KEY,
     useSecureSessionCookie: process.env.USE_SECURE_SESSION_COOKIES === 'true',
+    googleAnalyticsId: 'G-FFM8G1FWCZ',
```

Comment:
if this isn't going to be an environment variable it seems simpler just to keep it hardcoded in the script file. 

I don't really mind either way, just a comment.